File: 386-lang/homework2/fifa2014group.js
var request = require("request");
var groupArg = process.argv[2];

request("http://worldcup.kimonolabs.com/api/teams?apikey=CyrAZ7iUDOweTEqiMYgqI8ByZUDBtfiL&sort=group", 
function(err, response, body) {
	var req = JSON.parse(body)

	console.log("Name             " + "W  " + "D  " + "L");

	for (item in req) {
		if (req[item].group == groupArg) {
			var nameSpacing = (16 - req[item].name.length);
			var name = req[item].name;

			while (nameSpacing >= 0){
				name += " "
				nameSpacing--;
			}

			console.log(name + req[item].wins + "  " + req[item].draws + "  " + req[item].losses);
		}
	}
});

File: 386-lang/homework1/fifa2014group.py
import sys, requests, json

p = {'sort': 'group', 'apikey': 'CyrAZ7iUDOweTEqiMYgqI8ByZUDBtfiL'}
r = requests.get('http://worldcup.kimonolabs.com/api/teams', params=p)
group = sys.argv[1]
data = json.loads(r.content)

print "{0:<16} {1:^2} {2:^2} {3:^2}".format("Name", "W", "D", "L")

for item in data:
	if item["group"] == group:
		print "{0:16} {1:^2} {2:^2} {3:^2} ".format(item["name"], item["wins"], item["draws"], item["losses"])







